### PR TITLE
Allow Devs to attempt to run Frontend on Java 11

### PR DESCRIPTION
**Pull request by rtyley**
## What does this change?

In February 2015, we added a check with PR https://github.com/guardian/frontend/pull/8448 that asserted Frontend _had_ to run under Java 8 (many developers didn't have it yet). This change amends that check, to permit developers to run their code locally under Java 11 if they want to - Java 11 use is gaining traction at the Guardian (eg https://github.com/guardian/ophan/pull/4047) as it has much better performance on ARM than Java 8.

## What is the value of this and can you measure success?

I just want to allow myself to use Java 11 now that I've updated my dev box to have it as the default! This doesn't require that DotCom Frontend moves to running on Java 11 in PROD, or Java 11 in CI (though it probably would be a good idea!)  - I just want to allow casual developers who have to run Frontend as part of something else the luxury of not switching back to Java 8 if they don't have to!

### Tested

- [x] Locally  - minimum testing
- [ ] On CODE (optional)

I was able to run these sbt commands:

```
project dev-build
run
```

...and get an article page served, I didn't look much further than that!

### CODE REVIEW

File: common/app/services/S3.scala [hunk 1]
```diff
@@ -15,7 +15,6 @@ import conf.Configuration
 import model.PressedPageType
 import org.joda.time.{DateTime, DateTimeZone}
 import play.api.libs.ws.{WSClient, WSRequest}
-import sun.misc.BASE64Encoder
```
Comment (buck06191) [inline common/app/services/S3.scala:18]: Is this related to the Java 11 change?

**Review by buck06191 — APPROVED** [review]
LGTM, just want to check why the `BASE64Encoder` was removed in case it was in error as it seems (to my untrained eye) unrelated.

File: common/app/services/S3.scala (hunk 1 above)
Comment (rtyley) [reply common/app/services/S3.scala:18]: Yep! The import was actually unused as well (so the code compiles without it) but `sun.misc.BASE64Encoder` was an [internal, unsupported part of the JDK](https://www.oracle.com/java/technologies/faq-sun-packages.html), and went away with Java 9.
